src/vtp/ops/run_mock_election_operation.py

```diff
@@ -0,0 +1,474 @@
+                        self.parsed_args.printonly,
+                    )
+                )
+                tally_contests.main(the_election_config)
+                return
+            merge_contests = MergeContestsOperation(
+                Shellout.handle_printonly_switch(
+                    [
+                        "-r",
+                        "-m",
+                        self.parsed_args.minimum_cast_cache,
+                        "-v",
+                        self.parsed_args.verbosity,
+                    ],
+                    self.parsed_args.printonly,
+                )
+            )
+            merge_contests.main(the_election_config)
+            logging.info("Sleeping for 10")
+            time.sleep(10)
+            elapsed_time = time.time() - start_time
+            if elapsed_time > seconds:
+                break
+        if self.parsed_args.flush_mode in [1, 2]:
+            print("Cleaning up remaining unmerged ballots")
+            merge_contests = MergeContestsOperation(
+                Shellout.handle_printonly_switch(
+                    [
+                        "-r",
+                        "-f",
+                        "-v",
+                        self.parsed_args.verbosity,
+                    ],
+                    self.parsed_args.printonly,
+                )
+            )
+            merge_contests.main(the_election_config)
+        # tally the contests
+        tally_contests = TallyContestsOperation(
+            Shellout.handle_printonly_switch(
+                [
+                    "-v",
+                    self.parsed_args.verbosity,
+                ],
+                self.parsed_args.printonly,
+            )
+        )
+        tally_contests.main(the_election_config)
+
+    ################
+    # main
+    ################
+    # pylint: disable=duplicate-code
+    def main(self, the_election_config):
+        """Main function - see -h for more info
+
+        Note - this is a serial synchronous mock election loop.  A
+        parallel loop would have one VTP server git workspace somewhere
+        and N VTP scanner workspaces someplace else.  Depending on the
+        network topology, it is also possible to start up VTP scanner
+        workspaces on other machines as long as the git remotes and clones
+        are properly configured (with access etc).
+
+        While a mock election is running, it is also possible to use yet
+        another VTP scanner workspace to personally cast/insert individual
+        ballots for interactive purposes.
+
+        Assumes that each supplied town already has the blank ballots
+        generated and/or already committed.
+        """
+
+        # If an address was used, use that
+        if (
+            self.parsed_args.address
+            or self.parsed_args.state
+            or self.parsed_args.town
+            or self.parsed_args.substreet
+        ):
+            the_address = Address.create_address_from_args(
```

Comment:
This is for later, but since the argv string shouldn't be visible here, the address should be created from parameters to run.

```python
address = Address(address, state, town, substreet, ...)
```

Reply:
?  Even in the case that RunMockElectionOperation.run had the parameters passed into it, wouldn't the run function still be the place to create the address?  It would not want to be done in the argument parser function nor in the shell shim.  So, it would still be done here regardless if the information is in the object or as args to run.

I think this is also covered in the general open issue of where the argument parsing should go and to keep things simple, will resolve this conversation as well.